modular_skyrat/modules/encounters/code/nri_raiders.dm

```diff
@@ -1,3 +1,39 @@
+/// Possible places/departments to inspect
+#define INSPECTION_LIST pick( \
+	"Cargo", \
+	"Command", \
+	"Engineering", \
+	"Medical", \
+	"Science", \
+	"Service", \
+	"Security", \
+)
+
+/// List of possible decorative job titles for additional roleplay potential
+#define NRI_JOB_LIST pick( \
+	"NRI Patrol Officer", \
+	"NRI Satellite-Patrol Officer", \
+	"NRI Yefreitor-Patrol Officer", \
+	"NRI Junior Patrol Officer", \
+	"NRI Government Investigator", \
+	"NRI Field Inspector", \
+	"NRI Counter-Truancy Officer", \
+)
+
+/// List of possible decorative leader job titles for additional roleplay potential
+#define NRI_LEADER_JOB_LIST pick( \
+	"NRI Lieutenant Officer", \
+	"NRI Sergeant Officer", \
+	"NRI Senior Patrol Officer", \
+	"NRI Lead Investigator", \
+	"NRI Lead Inspector", \
+)
```

Comment:
I hadn't noticed that you were using pick directly here, that's not a good practice, it just hides parts of the logic in the define. Just make them lists and do the pick() on them in the code.
```suggestion
#define INSPECTION_LIST list( \
	"Cargo", \
	"Command", \
	"Engineering", \
	"Medical", \
	"Science", \
	"Service", \
	"Security", \
)

/// List of possible decorative job titles for additional roleplay potential
#define NRI_JOB_LIST list( \
	"NRI Patrol Officer", \
	"NRI Satellite-Patrol Officer", \
	"NRI Yefreitor-Patrol Officer", \
	"NRI Junior Patrol Officer", \
	"NRI Government Investigator", \
	"NRI Field Inspector", \
	"NRI Counter-Truancy Officer", \
)

/// List of possible decorative leader job titles for additional roleplay potential
#define NRI_LEADER_JOB_LIST list( \
	"NRI Lieutenant Officer", \
	"NRI Sergeant Officer", \
	"NRI Senior Patrol Officer", \
	"NRI Lead Investigator", \
	"NRI Lead Inspector", \
)
```